src/org/labkey/test/tests/AttachmentFieldTest.java

```diff
@@ -74,28 +75,27 @@ public void testFileFieldInSampleType()
         DataRegionTable samplesTable = DataRegionTable.DataRegion(getDriver()).withName("Material").waitFor();
         samplesTable.clickInsertNewRow();
         setFormElement(Locator.name("quf_Name"), "S1");
-        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_PDF);
+        setFormElement(Locator.name("quf_" + fieldName), SAMPLE_FILE);
         clickButton("Submit");
 
         log("Verifying view in browser works");
-        Locator.linkContainingText(SAMPLE_PDF.getName()).findElement(getDriver()).click();
-        log("Relative URL : " + getCurrentRelativeURL());
-        checker().verifyTrue("Incorrect PDF file displayed", getCurrentRelativeURL().contains("core-downloadFileLink.view"));
+        Locator.tagWithAttributeContaining("img", "title", SAMPLE_FILE.getName()).findElement(getDriver()).click();
```

Comment:
This click navigates. It should wait for navigation:
```suggestion
        clickAndWait(Locator.tagWithAttributeContaining("img", "title", SAMPLE_FILE.getName()));
```